tests/test_enums.py

```diff
@@ -21,8 +21,6 @@ def test_basics(self):
 
         cls = self.define_foobar()
 
-        self.assertIsInstance(cls, EnumType)
```

Comment:
Does this not still work?

Reply:
It does in my version. The commits pushed below have this. Sorry if this breaks CI. 😉